lib/ansible/modules/system/filesystem.py

```diff
@@ -184,6 +190,13 @@ def main():
             'grow_flag' : 'filesystem resize',
             'force_flag' : '-f',
             'fsinfo': 'btrfs',
+        },
+        'LVM2_member' : {
```

Comment:
not something changed in this PR, but I think this would be better as default matching the named passed in (most fs follow same pattern) and only put in entries for what needs to be overriden